ImageFeed/Service/Profile/ProfileImageService.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ProfileImageService.swift
+//  ImageFeed
+//
+//  Created by Filosuf on 28.11.2022.
+//
+
+import Foundation
+
+final class ProfileImageService {
+    // MARK: - Properties
+    static let shared = ProfileImageService()
+
+    static let didChangeNotification = Notification.Name(rawValue: "ProfileImageProviderDidChange")
+    private (set) var avatarURL: String?
```

Comment:
Обычно принято вот такое написание
```suggestion
    private(set) var avatarURL: String?
```